### PR TITLE
Added toast message for users to login on wishcards and homepage

### PR DESCRIPTION
**Pull request by deep-a**

### CODE REVIEW

File: client/views/home.ejs [hunk 1]
```diff
@@ -140,6 +140,14 @@
       $('.preloader').fadeOut(1500);
     });
   </script>
+<!--  this is for login toast -->
+  <script>
+    $(window).on('load', function() {
+      setTimeout(function() {<% if (!locals.user) { %>
```
Comment (Enubia) [inline client/views/home.ejs:146]: I'd move the script to an own file and just reference it from there if you need to call the script twice

**Review by Enubia — CHANGES_REQUESTED** [review]

**Comment by gnorbsl** [comment]
whats with all the changes in the package-lock? looks like a few packages are downgraded there.

can you get the latest master and run npm install

**Comment by stacysealky** [comment]
Is this all good to merge now? @deep-a @gnorbsl

**Comment by gnorbsl** [comment]
No it srill requires changes and he has to merge latest changes into his branch too

**Comment by deep-a** [comment]
@Enubia would something like this be better?
```
<% if (!locals.user) { %>
  <script src="/public/clientJS/loginToast.js"></script>
<% } %>
```

**Comment by Enubia** [comment]
please fix the package-lock file once more @deep-a

**Review by Enubia — APPROVED** [review]

**Review by gnorbsl — APPROVED** [review]

**Comment by stacysealky** [comment]
good to merge now? @Enubia @deep-a

**Pull request merged**